src/domain.test.ts

```diff
@@ -290,9 +299,13 @@ class FakeEditor implements Editor {
     return {
       theme: null,
       fontFamily: null,
-      previewMarkdownFiles: false
+      previewMarkdownFiles: false,
+      ...(this.slidesRcExist() && this._fakeRC)
```

Comment:
I prefer to stub the behavior of the FakeEditor in tests to keep the logic minimal and avoid having errors in tests because of the fake implementation. It also makes tests clearer to read IMO.

The FakeEditor is here to comply to the minimal required interface and behavior + make tests fast and reliable.